molecule/default/verify.yml

```
@@ -0,0 +1,20 @@
+---
+- name: Molecule | Verify | All
+  hosts: all
+
+  tasks:
+    - name: Molecule | Verify | Include all tests
+      ansible.builtin.include_tasks: "{{ item }}"
+      with_fileglob:
+        - ../tests/postgres/postgres.yml
+        - ../tests/patroni/*.yml
+        - ../tests/etcd/*.yml
```

```suggestion
        - ../tests/etcd/*.yml
        - ../tests/patroni/*.yml
        - ../tests/postgres/postgres.yml
```

it is worth checking etcd first and then patroni and only then postgres